src/Gui/DlgSettings3DView.ui

```diff
@@ -132,16 +133,16 @@ will be shown at the lower left in opened files</string>
       <item>
        <widget class="Gui::PrefCheckBox" name="CheckBox_useVBO">
         <property name="toolTip">
-         <string>If checked, Vertex Buffer Objects (VBO) will be used.
+         <string>If selected, Vertex Buffer Objects (VBO) will be used.
 A VBO is an OpenGL feature that provides methods for uploading
 vertex data (position, normal vector, color, etc.) to the graphics card.
 VBOs offer substantial performance gains because the data resides
 in the graphics memory rather than the system memory and so it
 can be rendered directly by GPU. 
 
-Note: Sometimes this feature may lead to a host of different issues ranging
-from graphical anomalies to GPU crash bugs. Remember to report this setting
-is enabled when seeking support on the FreeCAD forums.</string>
+&lt;b&gt;Note:&lt;/b&gt; Sometimes this feature may lead to a host of different
+issues ranging from graphical anomalies to GPU crash bugs. Remember to
```

Comment:
I disagree with adding markup like `<b>Note</b>`. Keep it simple.

Reply:
reverted in 92ea121